sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -111,6 +111,20 @@ class CmisApi(XcvrApi):
     LowPwrRequestSW = 4
     LowPwrAllowRequestHW = 6
 
+    LPO_HOST_ELECTRICAL_INTERFACE_IDS = [
```

Comment:
@longhuan-cisco which SFF spec version are you using?

Reply:
@longhuan-cisco @prgeor I corrected the LPO related IDs since this changes was causing test failures for https://github.com/sonic-net/sonic-platform-common/pull/596